pontoon/projects/tests/test_views.py

```diff
@@ -4,12 +4,17 @@
 from django.http import HttpResponse
 from django.shortcuts import render
 
-from pontoon.base.models import Project
-from pontoon.base.tests import (
-    ProjectFactory,
-    ResourceFactory,
-    TranslationFactory,
-)
+from pontoon.base.tests import TranslationFactory
+
+
+@pytest.mark.django_db
```

Comment:
I'd give this a https://pytest-cov.readthedocs.io/en/latest/markers-fixtures.html#no-cover, `@pytest.mark.no_cover`.

There's not enough assertions in the test, IMHO, to warrant counting this as test coverage. Doesn't make this a bad test, just not claim coverage for it.

Reply:
Good idea, thanks!